src/createSliderWithTooltip.jsx

```diff
@@ -26,14 +28,26 @@ export default function createSliderWithTooltip(Component) {
       });
     }
     handleWithTooltip = ({ value, dragging, index, disabled, ...restProps }) => {
-      const { tipFormatter } = this.props;
+      const {
+        tipFormatter,
+        tipProps,
+      } = this.props;
+
+      const {
+        prefixCls = 'rc-slider-tooltip',
+        overlay = tipFormatter(value),
+        placement = 'top',
+        ...restTooltipProps,
+      } = tipProps;
+
       return (
         <Tooltip
-          prefixCls="rc-slider-tooltip"
-          overlay={tipFormatter(value)}
           visible={!disabled && (this.state.visibles[index] || dragging)}
-          placement="top"
           key={index}
+          prefixCls={prefixCls}
+          overlay={overlay}
+          placement={placement}
+          {...restTooltipProps}
```

Comment:
@zeakd actually, many of our code rely on props ordering to control the props override logic